lib/node_modules/@stdlib/stats/base/dists/lognormal/cdf/benchmark/benchmark.js

```diff
@@ -32,17 +33,25 @@ var cdf = require( './../lib' );
 
 bench( pkg, function benchmark( b ) {
 	var sigma;
+	var len;
 	var mu;
 	var x;
 	var y;
 	var i;
 
+	len = 100;
+	x = new Float64Array( len );
+	mu = new Float64Array( len );
+	sigma = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		x[ i ] = uniform( -100.0, 100.0 );
```

Comment:
```suggestion
		x[ i ] = uniform( -100.0, 0.0 );
```

The range should remain the same as in the original, for consistency.